### PR TITLE
adding the form web page - the second part of task1

### PR DESCRIPTION
**Pull request by dohmeid**

### CODE REVIEW

File: index2.html
Comment (ehabAmriahExalt) [inline index2.html:1]: please rename the file to form.html

File: style2.css
Comment (ehabAmriahExalt) [inline style2.css:1]: please rename the file to form.css and move it under "css" folder

**Review by ehabAmriahExalt — CHANGES_REQUESTED** [review]
please check the comments

File: index2.html
Comment (dohmeid) [reply index2.html:1]: done

File: style2.css
Comment (dohmeid) [reply style2.css:1]: done

**Review by ehabAmriahExalt — APPROVED** [review]

**Pull request merged**